cfme/utils/appliance/__init__.py

```diff
@@ -1899,18 +1900,16 @@ def clean_appliance(self):
         self.ssh_client.run_command('service evmserverd stop')
         self.ssh_client.run_command('sync; sync; echo 3 > /proc/sys/vm/drop_caches')
         self.ssh_client.run_command('service collectd stop')
-        self.ssh_client.run_command('service {}-postgresql restart').format(
-            self.db.postgres_version)
+        self.ssh_client.run_command('service {}-postgresql restart'.format(
+            self.db.postgres_version))
         self.ssh_client.run_command(
-            'cd /var/www/miq/vmdb;DISABLE_DATABASE_ENVIRONMENT_CHECK=1 bin/rake evm:db:reset')
```

Comment:
Why is it removed?

Reply:
It was added to avoid an issue with 5.6, no need for now